packages/virtualized-lists/Lists/VirtualizedList.js

```diff
@@ -146,6 +146,11 @@ class VirtualizedList extends StateSafePureComponent<Props, State> {
     this.scrollToOffset({animated, offset});
   }
 
+  scrollToStart(params?: ?{animated?: ?boolean, ...}) {
+    const animated = params ? params.animated : true;
+    this.scrollToOffset({animated, offset: 0});
```

Comment:
how does this behave if ScrollView has insets?

Reply:
how I can add inserts?